src/app/index.js

```diff
@@ -11,6 +12,8 @@ import { InaccessibleContentError } from './errors.js';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 const SERVICE_DECLARATIONS_PATH = path.resolve(__dirname, '../../', config.get('serviceDeclarationsPath'));
+const MAX_PARALLEL_DOCUMENTS_TRACKS = 20;
+const MAX_PARALLEL_REFILTERS = 20;
```

Comment:
Yep I think it's better because even if it parallelization only improves by a little the refiltering time it improves significantly the tracking time. So the two numbers have different impacts